### DISCUSSION TITLE
Can I use Window Controls Overlay API?

### DISCUSSION THREAD

**Discussion by RyoTagami** [wailsapp/wails#2029] [Q&A] [answered]
In Wails, can I use [Window Controls Overlay API](https://github.com/WICG/window-controls-overlay/blob/main/explainer.md) like [electron's custom feature](https://www.electronjs.org/docs/latest/tutorial/window-customization#window-controls-overlay-macos-windows).

I would like to customize the title bar while keeping the close, maximize, and minimize buttons of the OS.

**Comment by leaanthony** [comment, marked as answer]
Not right now. Is the windows control overlay just a fake overlay or the real deal? I'm sure I've seen electron apps with shonky controls...
